vendor/github.com/automationbroker/bundle-lib/runtime/watch_pod_test.go

```diff
@@ -172,10 +171,6 @@ func TestWatchPod(t *testing.T) {
 					if newDescription != "" {
 						descriptions = append(descriptions, newDescription)
 					}
-
```

Comment:
Should this actually be removed?

Reply:
yeah I removed it because it wasn't used and lint was complaining. We would set it and did nothing with it.